### PR TITLE
FE-872: Component Wrapper for Matchbox Error

### PR DESCRIPTION
**Pull request by bkemper**
### What Changed

These are the components affected.

Screen Shot 2020-03-17 at 12 41 43 AM

### How To Test

- Pick a date range less than 7 days on any of the signals pages

Screen Shot 2020-03-18 at 12 33 29 AM

- Click to browse file system, then cancel to see required error on recipient list and suppression list.

Screen Shot 2020-03-18 at 12 35 39 AM

- Type some alert name on create alert page then click submit button
Screen Shot 2020-03-18 at 12 41 16 AM

### CODE REVIEW

**Comment by bkemper** [comment]
Rebased branch to see if these Codecov checks are more accurate.

**Comment by shrirupa** [comment]
Not sure what is different here but this error message looks a little off. 
Screen Shot 2020-03-18 at 11 50 54 AM

File: src/components/matchbox/tests/Error.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { useHibana } from 'src/context/HibanaContext';
+import Error from '../Error';
+
+jest.mock('src/context/HibanaContext');
+// jest.mock('@sparkpost/matchbox-hibana', () => ({
+//   Error: props => <div data-id="hibana-inline" {...props} />,
```
Comment (shrirupa) [inline src/components/matchbox/tests/Error.test.js]: comments can be removed.

File: src/components/matchbox/tests/Error.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    );
+  };
+
+  it('renders the Hibana version of the Error component correctly when hibana is enabled', () => {
+    useHibana.mockImplementationOnce(() => [{ isHibanaEnabled: true }]);
+
+    const wrapper = subject();
+
+    expect(wrapper).toHaveProp('padding', '600');
+    expect(wrapper).toHaveDisplayName('HibanaError');
```
Comment (shrirupa) [inline src/components/matchbox/tests/Error.test.js]: 👍

**Comment by bkemper** [comment]
@shrirupa I just check on staging, this form doesn't use the label prop.

Screen Shot 2020-03-18 at 1 59 07 PM

**Pull request merged**